extensions/sql-database-projects/src/dialogs/publishDatabaseDialog.ts

```diff
@@ -1002,3 +997,15 @@ export function promptForPublishProfile(defaultPath: string): Thenable<vscode.Ur
 		}
 	);
 }
+
+export function promptToSaveProfile(project: Project, publishProfileUri?: vscode.Uri) {
```

Comment:
might make more sense for this function to move to `publishProfile.ts` with the other publish profile helper functions since it's being used by the quickpick too